spec.md

```diff
@@ -23,42 +23,25 @@ A standard token interface is a basic building block for many applications on th
    	totalSupply : Nat; // token total supply
    	owner : Principal; // token owner
    	fee : Nat; // fee for update calls
-   	feeTo : Principal; // fee receiver
    }
    ```
 
-2. Status: token status info
-
-   ```js
```

Comment:
I don't feel that deployTime, cycles and holderNumber are particularly important. I feel we would require implementations a little bit too much.